spring-cloud-consul-core/src/main/java/org/springframework/cloud/consul/ConsulAutoConfiguration.java

```diff
@@ -11,6 +13,7 @@
  */
 @Configuration
 @EnableConfigurationProperties
+@ConditionalOnProperty(name = "consul.enabled", matchIfMissing = true)
```

Comment:
I think we should do `spring.cloud.consul`.  We would need to make sure it is consistent throughout the project.